### PR TITLE
Fix scheme issue with loading private blog images

### LINKED ISSUE #3510: Private Blog images sometimes aren't loaded in the post preview
Sometimes images from private sites aren't being loaded in the post list preview's `WebView` (only for locally changed posts).

wpandroid-private-blog-image

### PR DESCRIPTION
**Pull request by aforcier**
Fixes #3510 by forcing `HTTPS` usage for images in private sites. They were failing to load when a post included images using the `HTTP` scheme, returning a 301 redirect instead of retrieving the image.

The bug was only occurring in the post list preview, but I applied the change to other areas in the app where resource requests are made just in case.

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/networking/WPDelayedHurlStack.java [hunk 1]
```diff
@@ -98,8 +99,10 @@ public HttpResponse performRequest(Request<?> request, Map<String, String> addit
                 additionalHeaders.put("Authorization", auth);
             }
 
+            String blogUriScheme = WordPress.currentBlog.getUri().getScheme();
```
Comment (maxme) [inline WordPress/src/main/java/org/wordpress/android/networking/WPDelayedHurlStack.java]: We should use `mCurrentBlog`, `WPDelayedHurlStack` could have been instantiated with another Blog.

Comment (aforcier) [inline WordPress/src/main/java/org/wordpress/android/networking/WPDelayedHurlStack.java]: Hmm `mCurrentBlog` was always null in my testing. I'll investigate this further.

**Comment by aforcier** [comment]
Per our conversation, I dropped the `blogUriScheme.equals("https")` check from `WPDelayedHurlStack` since the `WordPress.currentBlog` check is inaccurate and all of WordPress.com is using SSL anyway.

**Comment by maxme** [comment]
Looks good! :shipit:

**Comment by maxme** [comment]
Merged in `release/4.9` and `develop`

**Pull request merged**